README.md

```diff
@@ -1,6 +1,7 @@
 # Action
 
 [![Slack Status](http://slackin.parabol.co/badge.svg)](http://slackin.parabol.co/)
+[![Circle CI](https://img.shields.io/circleci/project/parabol/action/master.svg)](https://circleci.com/gh/ParabolInc/action)
```

Comment:
that uhh, that doesn't exist yet. 😋  
but feel free to sign up & get it going!
...although i'd wait until we're stable enough to begin linting, because there's probably 3+ hours of linting that needs to be done...